res/controllers/common-hid-packet-parser.js

```diff
@@ -652,26 +652,22 @@ HIDPacket.prototype.send = function() {
     }
 
     for (var group_name in this.groups) {
-        var group = this.groups[group_name];
+        const group = this.groups[group_name]
```

Comment:
While `const` somehow works without throwing any error in the old QtScript interpeter, I don't think it's actually `const`. To avoid confusion, please use `var` until we have ES7 available after merging #1795.

Reply:
const is used a lot throughout the scripts though, just did a quick search. Should this all be var then?

Reply:
ah this is all in the novation-launchpad thing - nevermind